doc/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	add_executable (markdownlinkconverter markdownlinkconverter/markdownlinkconverter.c)
+	find_util (markdownlinkconverter MARKDOWN_LINK_CONVERTER "")
+
```

Comment:
> So if I would use the mingw gcc to generate the docu it will fail?

Yes, it would call `wine` without your program as argument.

It is possible to workaround this argument limitation. We could generate a shell script which calls `wine markdownlinkconverter`. So better leave the usage of `find_util`: then we have a reminder that this place needs to be modified when we adapt `find_util` to work without arguments.